app/pages/project/index.cjsx

```diff
@@ -230,7 +240,7 @@ ProjectPageController = React.createClass
 
   checkUserRoles: (project, user) ->
     currentUserRoleSets = @state.projectRoles.filter((roleSet) => roleSet.links.owner.id is user.id)
```

Comment:
This line will still need an existential check on the user: `user?.id`. Trying the workflow query param when logged out is broken otherwise.